### DISCUSSION TITLE
props로 styles를 넘겨주는 것에 대한 생각

### DISCUSSION THREAD

**Discussion by igy95** [woowacourse/react-payments#47] [토론장]
컴포넌트의 추상화 수준이 높을수록 style에 대한 분기 처리?의 필요성도 같이 높아진다고 생각하는데 어떠한 컴포넌트의 style을 처리해줄 때 다들 어떤 방법을 쓰시는지, 혹시 props으로 style을 보내준다면 어느 정도까지 보내주는지 궁금합니다..!

**Comment by zereight** [comment]
prop을 직접 넣어주는 경우도 있고, ThemeProvider로 전역으로 넣어주는 경우도 있었어요.
보통 컬러, 사이즈 등을 넘겨주었던것 같습니당.
그리고 스타일에 관련된 인자는 $를 붙여주어서 구별하면 좋다라는 글도 어디선가 본것같아요
